packages/ui/components/ui/tx/view/asset-icon/unbonding-token-icon.tsx

```diff
@@ -0,0 +1,105 @@
+import { assetPatterns } from '@penumbra-zone/constants/src/assets';
+
+const getFirstEightCharactersOfValidatorId = (displayDenom = ''): [string, string] => {
+  const id = (assetPatterns.unbondingToken.capture(displayDenom)?.id ?? '').substring(0, 8);
+
+  const firstFour = id.substring(0, 4);
+  const lastFour = id.substring(4);
+
+  return [firstFour, lastFour];
+};
+
+export const UnbondingTokenIcon = ({
+  className,
+  displayDenom,
+}: {
+  className?: string;
+  displayDenom?: string;
+}) => {
+  const [firstFour, lastFour] = getFirstEightCharactersOfValidatorId(displayDenom);
+
+  return (
+    <svg
```

Comment:
This SVG markup feels a bit redundant right? Can this be combined with the other and we just need to pass a flag for the arrow direction + text color?

Reply:
Definitely. I should have left a comment about this — I was thinking of doing that as part of this PR, but then realized I should wait until design approval to do that, since it may be a waste of effort.